resources/refund.js

```diff
@@ -22,4 +22,20 @@ function Refund(shopify) {
 
 assign(Refund.prototype, pick(baseChild, ['get', 'buildUrl']));
 
+
+/**
+ * Calculates a refund.
+ * https://help.shopify.com/api/reference/refund#calculate
+ *
+ * @param {Number} parentId Parent record ID
+ * @param {Object} [params] Query parameters
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Refund.prototype.calculate = function complete(orderId, params) {
+  const url = this.buildUrl(orderId, '/calculate');
```

Comment:
Can you please remove the `/` from `'/calculate'`? It doesn't cause any harm but it's redundant.